aria/cli/commands/service_templates.py

```diff
@@ -181,21 +184,49 @@ def validate(service_template, service_template_filename,
     logger.info('Service template validated successfully')
 
 
+@service_templates.command(name='display',
+                           short_help='Display service template information')
+@aria.argument('service-template-name')
+@aria.options.verbose()
+@aria.options.display_json
+@aria.options.display_yaml
+@aria.options.display_types
+@aria.pass_model_storage
+@aria.pass_logger
+def display(service_template_name, model_storage, json, yaml, types, logger):
```

Comment:
Having both `display` and `show` is confusing :/
I agree however that it doesn't make sense to make this command into a mere flag of `show` as they're very different.
I'm generally ok with this as is, but Maxim wanted me to suggest `aria parse service/service-template` as an option as well. I'll leave it for your consideration :)